### PR TITLE
include iconic

### PR DESCRIPTION
**Pull request by ping1z**
include iconic

### CODE REVIEW

**Comment by ping1z** [comment]
iconic doc: https://useiconic.com/open/

File: src/assets.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import './styles/iconic/css//open-iconic-bootstrap.min.css';
+import './index.css';
```
Comment (probli) [inline src/assets.js:2]: each file should end with an empty line from [https://github.com/airbnb/javascript#whitespace--newline-at-end](url)

File: src/index.js [hunk 2]
```diff
@@ -4,34 +4,33 @@ import { Provider } from 'react-redux';
...
-import Author from './components/Author.js';
-import Reader from './components/Reader.js';
-import Home from './components/Home.js';
+import './assets.js';
 
+import Search from './components/search';
+import Article from './components/article';
+import Author from './components/author';
+import Reader from './components/reader';
+import Home from './components/home';
```
Comment (probli) [inline src/index.js:13]: filename should match the export name. https://github.com/airbnb/javascript#naming--filename-matches-export

Not necessary, but let's keep consistent with this style guidance.
https://github.com/airbnb/javascript/blob/master/react/README.md

File: src/index.js [hunk 3]
```diff
@@ -4,34 +4,33 @@ import { Provider } from 'react-redux';
...
 import registerServiceWorker from './registerServiceWorker';
 
 import reducers from './reducers';
 
-let store = createStore(reducers);
+let store = createStore(reducers)
```
Comment (probli) [inline src/index.js:19]: ";" needed.

File: src/index.js [hunk 4]
```diff
@@ -4,34 +4,33 @@ import { Provider } from 'react-redux';
...
+        <Switch>
+          <Route path="/search/" component={ Search } />
+          <Route path="/Article/:id" component={ Article } />
+          <Route path="/Author/:id" component={ Author } />
+          <Route path="/Reader/:id" component={ Reader } />
+          <Route path="/" component={ Home } />
+        </Switch>
+      </div>
+    </BrowserRouter>
+  </Provider>
```
Comment (probli) [inline src/index.js:33]: I made a mistake here. No spaces needed for braces in JSX.
https://github.com/airbnb/javascript/blob/master/react/README.md#spacing

**Comment by ping1z** [comment]
@lidaxia2017 please merge this commit first, then we do the styles change in from the master branch.

**Pull request merged**